activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -18,6 +18,9 @@ def visit_AddColumn(o)
           "ADD #{accept(o)}"
         end
 
+        delegate :quote_column_name, :quote_table_name, :quote_default_expression, :type_to_sql, to: :@conn
```

Comment:
If you just move this below the `private` statement, do you still need separate line to mark these as private?